prow/repoowners/repoowners.go

```diff
@@ -736,8 +737,35 @@ func (o *RepoOwners) applyOptionsToPath(path string, opts dirOptions) {
 	}
 }
 
-func (o *RepoOwners) filterCollaborators(toKeep []github.User) *RepoOwners {
-	collabs := sets.New[string]()
+// Extend github team to appproves/reviewers list
+// If appproves/reviewers name contains '/', like 'myorg/developer'
+// It will be recognized as github team.
+func extendGithubTeam(ars sets.String, ghc githubClient) sets.String {
```

Comment:
what is `ars`?

Also not too happy with the method name itself, could be `resolveGitHubTeamMembers`or something similar that hints that it returns members.

Also^2, I think  should now use generic sets in new code, the old ones are deprecated